File: Baekjoon/python/19945.py
# source : https://www.acmicpc.net/problem/19945

def main():
    # input
    n = int(input())

    if n >= 0:
        print(len(bin(n)[2:]))
    else:
        print(32)

if __name__ == '__main__':
    main()

File: Baekjoon/go/19945.go
// source : https://www.acmicpc.net/problem/19945

package main

import (
	"fmt"
	"strconv"
)

func main() {
	var n int

	// input
	fmt.Scan(&n)

	if n >= 0 {
		fmt.Println(len(strconv.FormatInt(int64(n), 2)))
	} else {
		fmt.Println(32)
	}
}